server/login.go

```diff
@@ -35,6 +35,20 @@ func (server *LoginServer) Initialise(dbuser, dbpassword, dbaddress, dbport, dbd
 	}
 
 	log.Println("Connected to database")
+
+	server.CleanupDB()
+
+	log.Println("Cleaned up the database")
+}
+
+// CleanupDB sets all accounts isLogedIn to 0
+func (server *LoginServer) CleanupDB() {
+	res, err := server.db.Exec("UPDATE `accounts` SET `isLogedIn`=0 WHERE `isLogedIn` = 1;")
+	if err != nil {
+		log.Println(err)
```

Comment:
Should probably terminate the application as this is a pretty bad error

Reply:
Changed to log.Fatal()